activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -4,6 +4,23 @@
 
 module I18n
   class Railtie < Rails::Railtie
+    module Helpers
+      def self.glob_to_exts(glob)
+        match = glob.match(/\{([\S]+)\}/)
+        match[1].split(",")
+      end
```

Comment:
Would probably rewrite as:

```ruby
def self.extensions_for(glob)
  $1.split(',') if glob =~ /\{([\S]+)\}/
end
```

Don't know if extensions is the right word, but I'd definitely want to write out whatever `exts` means.